[ADD] new module_pos_pricelist_technical: (#790)
OCA/pos: pos_pricelist_technical/readme/DESCRIPTION.rst

> This module extends the functionality of point of sale to prevent
> to select in the point of sale, technical pricelists that used for the
> computation of other pricelists, but that should not be selected.

Reviewer: ```suggestion
This module extends the functionality of Point of Sale to prevent
technical pricelists that are used for the
computation of other pricelists from being selected.
```
